src/ui/FacetSlider/FacetSlider.ts

```diff
@@ -618,7 +618,7 @@ export class FacetSlider extends Component {
       {
         className: 'coveo-facet-slider-breadcrumb-clear'
       },
-      SVGIcons.icons.checkboxHookExclusionMore
```

Comment:
Is the old icon used anywhere else? or could we remove it from the project?

Reply:
It's used for facets